mobile/src/modules/auth/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { StyleSheet, View, Picker, Button } from 'react-native';
+import { NavigationInjectedProps, withNavigation } from 'react-navigation';
+import { connect } from 'react-redux';
+import { Text, TextThemes } from '../../../core/components/base/Text';
+import { HeaderTitle } from '../../../core/components/header/HeaderTitle';
+import { i18n } from '../../../core/i18n';
+import { InjectedReduxProps } from '../../../core/interfaces';
+import { FullHeightView } from '../../../core/layout/FullHeightView';
+import { Screen } from '../../../core/layout/Screen';
+import { ApplicationState } from '../../../core/store/initialState';
+import { Sizes } from '../../../core/theme/sizes';
+import { AuthReduxState } from '../store/reducer';
+import { core } from '../translations';
+import { routes } from '../../../core/navigation/routes';
+
+interface IProps extends InjectedReduxProps {
+    auth: AuthReduxState,
+}
+type Props = IProps & NavigationInjectedProps;
+
+const styles = StyleSheet.create({
+    hintView: {
+        paddingTop: Sizes.SMALL,
+        flexGrow: 1
+    }
+});
+
+class Home extends React.PureComponent<Props> {
+    static navigationOptions = {
+        headerTitle: <HeaderTitle>{i18n.t(core.screens.home.title)}</HeaderTitle>
+    };
+
+    changeAccount() {
+        this.props.navigation.navigate(routes.accounts);
+    }
+
+    render() {
+        const selectedAccount = this.props.auth.accounts.length && 
```

Comment:
I prefer just use `!isEmpty()` from `lodash`;
And i think if we have no accounts, then we shouldn't show "fake" balances or so, why not to show button to create accounts or forward to accounts screen in `componentDidMount` hook

Reply:
Yes this is temp code - will be much more elegant once we move selectedAccount to the reducer.